### PR TITLE
test: add cli tests

### PR DESCRIPTION
**Pull request by alopezsanchez**

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -29,6 +22,9 @@ class CLI {
   constructor(stream) {
     this.stream = stream || process.stderr;
     this.spinner = ora({ stream });
+    this.SPINNER_STATUS = SPINNER_STATUS;
+    this.SCISSOR_LEFT = '>8';
```
Comment (alopezsanchez) [inline lib/cli.js]: Are we using the SCISSORS?

File: lib/figures.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const chalk = require('chalk');
+const {
+  tick, cross, info: infoRaw, warning: warningRaw
+} = require('figures');
```
Comment (addaleax) [inline lib/figures.js]: Do we need to add this to package.json?

File: test/unit/cli.test.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        assert.strictEqual(stream.toString(), `${info}  Info!${EOL}`);
+      });
+
+      it('should print an info message in a new line if specified', () => {
+        cli.info('Info!', { newline: true });
+        assert.strictEqual(stream.toString(), `${EOL}${info}  Info!${EOL}`);
+      });
+    });
+
+    // TODO: `Error` instance test
```
Comment (alopezsanchez) [inline test/unit/cli.test.js]: Can you please give me an example of an`Error` instance with `data` property?

File: lib/figures.js (hunk 2 above)
Comment (alopezsanchez) [reply lib/figures.js]: `figures` is already a dependency.

File: lib/cli.js (hunk 1 above)
Comment (cPhost) [reply lib/cli.js]: I dont think we are using them.

File: test/unit/cli.test.js (hunk 3 above)
Comment (joyeecheung) [reply test/unit/cli.test.js]: I think you can get that by `console.log(JSON.stringify(err))` and try pulling a PR id that's actually an issue

File: lib/cli.js (hunk 1 above)
Comment (joyeecheung) [reply lib/cli.js]: Yes they can be removed now.

**Comment by joyeecheung** [comment]
@alopezsanchez Appveyor failed: https://ci.appveyor.com/project/joyeecheung/node-core-utils/build/205.master

**Comment by alopezsanchez** [comment]
@joyeecheung fixed.
https://travis-ci.org/nodejs/node-core-utils/builds/305526482
https://ci.appveyor.com/project/joyeecheung/node-core-utils/build/213.master

**Pull request merged**